.packit.yaml

```
@@ -0,0 +1,286 @@
+# Default packit instance is a prod and only this is used
+# stg instance is present for testing new packit features in forked repositories where stg is installed.
+packit_instances: ["prod", "stg"]
+upstream_project_url: https://github.com/debezium/debezium
+issue_repository: https://github.com/debezium/debezium
+jobs:
+  - job: tests
+    trigger: pull_request
+    # Suffix for job name
+    identifier: "postgres-12-decoderbufs"
+    targets:
+      - centos-stream-8-x86_64
+    skip_build: true
+    manual_trigger: true
+    labels:
+      - postgres
+    tf_extra_params:
+      test:
+        fmf:
```

```suggestion
        tmt:
```

Hi, sorry for the trouble, Testing Farm is deprecating the `test.fmf` in favor of `test.tmt` and we have already switched to it. You should get it running by switching here and below to `tmt`.

---

Thanks, done. Seems to be ok now